api_tests/e2e/rfc003/btc_eth/test.js

```diff
@@ -28,11 +32,12 @@ const beta_expiry = new Date("2080-06-11T13:00:00Z").getTime() / 1000;
 describe("RFC003: Bitcoin for Ether", () => {
     before(async function() {
         this.timeout(5000);
-        await bitcoin.btc_activate_segwit();
+        await bitcoin.activateSegwit();
+        // Initialize the client
```

Comment:
:question: Why is this comment here? It doesn't seem to be there in other tests :D 
```suggestion
```

Reply:
Let's remove for now, it's a future clean up.